### PR TITLE
Update about me to include Neil's fave website

### PR DESCRIPTION
**Pull request by ericaevans**
I couldn't think of my own favorite website, so I just included Neil's to the default about me page.

Does that mean I don't have my own personality? What do you think @SparkleCapps

### CODE REVIEW

File: about-me.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 ![Picture of me]()
 
-Hi, I'm Erica Evans. I'm a person at *USDS*.
+Hi, I'm Erica Evans. I'm a member of the procuremenati at *USDS*.
```
Comment (sparkle-usds) [inline about-me.md:5]: Love the updates here!

Comment (ericaevans) [reply about-me.md:5]: Thanks

Comment (sparkle-usds) [reply about-me.md:5]: procuremenati is the best!!

Comment (sparkle-usds) [reply about-me.md:5]: merge away!

**Pull request merged**